api/src/main/java/org/pmiops/workbench/api/CohortsController.java

```diff
@@ -135,6 +140,12 @@ public Cohort apply(org.pmiops.workbench.db.model.Cohort cohort) {
       String cohortId, Cohort cohort) {
     org.pmiops.workbench.db.model.Cohort dbCohort = getDbCohort(workspaceNamespace, workspaceId,
         cohortId);
+    if(!Strings.isNullOrEmpty(cohort.getEtag())) {
```

Comment:
We should probably discuss over Slack whether not specifying an etag should be allowed.

My initial leaning is it should be required; we don't want somebody to forget and have data get clobbered.

Reply:
Sounds fine to not include it in this PR, but once the UI is updated I'd start enforcing that etags are required, until such time as we have a use case where they shouldn't be... that can hopefully help us avoid bugs.